keyboards/zlabkeeb/vrynboard/config.h

```diff
@@ -0,0 +1,25 @@
+/*
+Copyright 2024 zlabkeeb (zlabkeeb@gmail.com)
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#define OLED_TIMEOUT 0
```

Comment:
Just a heads up, I don't think that this is a good idea, as you're not handling the timeout yourself.  Leaving the OLED enabled the entire time can/will lead to burn-in on the screen. 

Additionally, the oled timeout uses a 32-bit unsigned integer, and is in ms. That means that the max value can handle almost 50 days. 

If the 60 seconds for the default isn't long enough, then I would recommend something longer, but still a defined value.

Reply:
Alright, thanks for the heads-up. I'll adjust the OLED timeout accordingly.